src/conversions/sample_rate.rs

```diff
@@ -83,10 +77,13 @@ where
             (first, next)
         };
 
+        // Reducing nominator to avoid numeric overflows during interpolation.
+        let (to, from) = Ratio::new(to, from).into_raw();
```

Comment:
I took a look at the Ratio source, and this does the same as the original code that did `from: from/gcd` and `to: to/gcd`. See: https://docs.rs/num-rational/latest/src/num_rational/lib.rs.html#108-112. Though I had now idea why they did that until now.

Given that what is the motivation for using `Ratio` over `/ gcd`?

Reply:
> Even if this is an extra call it happens only once at the converter creation.

pretty sure that will be in-lined so i'll be okay

> It is also used in tests, to limit input values, I thought it makes sense to re-use that and have a bit less code to test.

:+1: